### ISSUE TITLE
define entry point for yagni

### ISSUE THREAD

**Issue by kurt-o-sys** [tolitius/boot-check#3]
yagni uses :main in a leiningen project to find the entry point. This entry point is skipped in the yagni analysis. However, in using boot and boot-check, this entry point is not found by yagni and there's no way to add a :main entry to boot-check (adding a :main to the jar task-options isn't really helpful here). 
This results always in:

```
 WARN: could not find any references to the following:

<project>/-main
```

or something similar. 

It is possible to add the main entry point in .lein-yagni, but this is somewhat confusing as one is using boot and needs to add a file which clearly points to leiningen. It would make much more sense to be able to add the entry points inside build.boot, or in .boot-yagni or something similar.

**Comment by tolitius** [comment]
sure, the `args` part of all the checkers is a `TO DO` including yagni.

I'll add args to the [yagni task](https://github.com/tolitius/boot-check/blob/master/src/tolitius/boot_check.clj#L46), so it'd be possible to pass `main` entry points to `with-yagni`

**Comment by tolitius** [comment]
@qsys I added `options` to yagni. check out the [test run](https://github.com/tolitius/boot-check/blob/227ea48bec5ca53363d8cd8d55f8c96b619bd625/build.boot#L21-L23).

i.e.

``` clojure
(check/with-yagni :options {:entry-points ["app.foo/-main"
                                           "app.bar/fun"
                                           "app.baz/another-entry-point"]})))
```

try it out with `0.1.3-SNAPSHOT`, let me know if it works for you.

**Comment by tolitius** [comment]
I released `boot-check` `0.1.3` with this feature.

feel free to reopen this in case you have any more questions.

**Comment by kurt-o-sys** [comment]
Great, thanks! It works :)
